### DISCUSSION TITLE
Column burden and surface emissions in ADF

### DISCUSSION THREAD

**Discussion by adagj** [NorESMhub/noresm3_dev_simulations#54] [Oslo-Aero]
Hi,
I have added (tried at least) aerosol column burden and surface emissions to the ADF table. Here is a comparison between n1850.ne30_tn14.hybrid_fatessp.202401007 and NorESM2-MM. Can I get some feedback on the results, and if the values look reasonable? 

https://ns2345k.web.sigma2.no/datalake/diagnostics/ADF/plots/n1850.ne30_tn14.hybrid_fatessp.202401007_5_10_vs_N1850frc2_f09_tn14_20191001_1204_1209/website/html_table/amwg_table_Case%20Comparison.html
 
There is also a new version of ADF with chemistry diagnostics which we probably want to test: Using ADF chemistry diagnostics: https://wiki.ucar.edu/display/camchem/Model+Diagnostics+and+Evaluation+with+ADF

@oyvindseland @DirkOlivie @MichaelSchulzMETNO @andrearosendahl

**Comment by MichaelSchulzMETNO** [comment]
This is great! I have suggested to Andrea, that we plot with pyaerocom/aeroval in parallel the maps of several key old simulations (CMIP6 MM PI + PD , CMIP6 LM PI +PD, Gliss et al AeroCom CAM5.3, plus the latest 2.5  PI). For comparison we also have the old [AeroCom website ](https://aerocom.met.no/cgi-bin/surfobs_annualrs.pl?PROJECT=NorESM&MODELLIST=NorESM-Reference&FULL=explicit&INFO=nohover&PERFORMANCE=ind&YEARFILTER=ALLYEARS&PSFILTER=ALLVARS&Type0=MAP&Run0=CAM4-Oslo-Vcmip5.A2.CTRL&Parameter0=LOAD_BC&Station0=WORLD&Year0=an9999&Period0=mALLYEAR) - but it will take a little time to get it all right....

**Comment by MichaelSchulzMETNO** [comment]
<img width="1102" alt="Screenshot 2024-11-06 at 11 17 07" src="https://github.com/user-attachments/assets/f6a29604-4449-4d78-a710-f2d857081eba">

**Comment by MichaelSchulzMETNO** [comment]
comment to my comparison table - - I think this looks all pretty correct in ADF !

**Reply by adagj** [reply]
Thanks for checking @MichaelSchulzMETNO ! Dirk also confirmed it was "pretty" correct :-)
I'll push my hacks to the ADF fork then :)

**Comment by oyvindseland** [comment]
It is also quite close to the ncl results so looks fine.

I think the ncl  makes a simplification by averaging over the months, i.e. no variation in the number of days.
https://ns2345k.web.sigma2.no/datalake/oyvinds/ModIvsModII/n1850.ne30_tn14.hybrid_fatessp.202401007vsN1850frc2_f09_tn14_20191001/yr5-10/ModIvsModII_B.htm

The latitude weights are now named gw again in the model output files. P0 is still missing
